types/three/three-core.d.ts

```diff
@@ -3278,9 +3278,7 @@ export interface HSL {
  * @see <a href="https://github.com/mrdoob/three.js/blob/master/src/math/Color.js">src/math/Color.js</a>
  */
 export class Color {
-    constructor(color?: Color);
-    constructor(color?: string);
-    constructor(color?: number);
+    constructor(color?: Color|string|number);
```

Comment:
Just curious I assume we prefer union to overload to make it possible to actually pass a union?

Reply:
I'm working a gaming project, and I'm going to work with many config file.Many config file uses either string or number to declare a color.So I need a function like this:

```javascript
function createColor(colorParam: string|number){
    return new Color(color);
}
```

this function cannot work.And I don't want this function like
```javascript
function createColor(colorParam:string|number){
    if(typeof colorParam === "string"){
        return new Color(color);
    }else if(typeof colorParam === "number"){
        return new Color(color)
    }
}
```